README.md

```diff
@@ -18,6 +18,8 @@ The template document is made up of variants, prefs and values.
 _**pref**_ is the name of the preference that will be added to the prefs.js file. This is an unquoted string.
 Valid prefs can be found in [all.js](https://hg.mozilla.org/mozilla-central/file/tip/modules/libpref/init/all.js) or in [StaticPrefList.yml](https://hg.mozilla.org/mozilla-central/file/tip/modules/libpref/init/StaticPrefList.yaml).
 
+__**review_on_close**__ is optional. It is list of relevant Bugzilla IDs used to help avoid collecting obsolete entries. When a bug in the list is closed the entry will be reviewed and removed if appropriate.
```

Comment:
Should this read, `When all bugs in the list`?  We never discussed the behavior but I imagine if we're accepting a list, we don't want to remove the pref until all entries are resolved.

Reply:
Yes that makes more sense.